frontend/src/views/admin/matching/index.tsx

```diff
@@ -1,14 +1,76 @@
 import React from "react";
+
 import { ContentArea } from "../../../components/layout";
 import { useSelector } from "react-redux";
 import { activeSessionSelector, fetchPostings } from "../../../api/actions";
 import { useThunkDispatch } from "../../../libs/thunk-dispatch";
-import { GroupedOverview } from "./grouped-overview";
+import { round } from "../../../libs/utils";
+
+import {
+    applicationsSelector,
+    assignmentsSelector,
+    positionsSelector,
+    applicantsSelector,
+} from "../../../api/actions";
+import { Assignment, Application, Applicant } from "../../../api/defs/types";
+
+import {
+    matchesSelector,
+    guaranteesSelector,
+    batchUpsertMatches,
+} from "./actions";
+import { PositionSummary, ApplicantSummary, Match } from "./types";
+
+import { PositionList } from "./position-list";
+import { ApplicantView } from "./applicant-view";
+
+import {
+    ImportMatchingDataButton,
+    ImportGuaranteesButton,
+    ExportMatchingDataButton,
+} from "./import-export";
+import { FinalizeChangesButton } from "./finalize-changes";
+
+function GetNewestApplication(
```

Comment:
Is this a utility function or a React component? Only React components should start with a capital letter. Utility functions should always start with a lowercase.